engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/StreamProcessorReplayModeTest.java

```diff
@@ -109,6 +110,40 @@ public void shouldReplayContinuously() {
     assertThat(getCurrentPhase(replayContinuously)).isEqualTo(Phase.REPROCESSING);
   }
 
+  @Test
+  public void shouldRecoverAndReplayCorrectlyWhenEventAtSnapshotPositionIsNotAvailableOnStart() {
```

Comment:
:upside_down_face: The name of the test is too long to read. 

We could rename it to something shorter like `shouldReplayIfNoEventsAfterSnapshot`.